h2o-core/src/main/java/water/jdbc/SQLManager.java

```diff
@@ -276,8 +278,10 @@ static String buildSelectChunkSql(String databaseType, String table, long start,
 
     String sqlText = "SELECT " + columns + " FROM " + table;
     switch(databaseType) {
+      case SQL_SERVER_DB_TYPE:  // requires ORDER BY clause with OFFSET/FETCH NEXT clauses, syntax supported since SQLServer 2012
+        sqlText += " ORDER BY ROW_NUMBER() OVER (ORDER BY (SELECT 0))";
+        //fall through oracle syntax for offset logic
```

Comment:
this is will work, do you think there is a chance someone can broke SQL Server by modifying Oracle statement? Maybe it would be better to duplicate instead. WDYT?

Reply:
yeah, was hesitating... probably safer to duplicate, it's just one line after all

Reply:
agree, better to duplicate and not mix sql flavors